client/app/api/course/Assessment/Monitoring.ts

```diff
@@ -0,0 +1,17 @@
+import { MonitoringRequestData } from 'types/course/assessment/monitoring';
+
+import { APIResponse } from 'api/types';
+
+import BaseAssessmentAPI from './Base';
+
+class MonitoringAPI extends BaseAssessmentAPI {
+  get #urlPrefix(): string {
+    return `/courses/${this.courseId}/assessments/${this.assessmentId}/monitoring`;
+  }
+
+  index(): APIResponse<MonitoringRequestData> {
```

Comment:
Perhaps include this in `client/app/api/course/Assessment/Assessments.js` since the monitoring endpoint is declared within the assessment resource/controller?